lib/l10n/app_en.arb

```diff
@@ -350,5 +335,30 @@
   "pageChatNoChatMessage": "You can chat with your driver as soon as your ride has been accepted",
   "pageChatMessageBarHint": "Write a Message",
 
+  "pageHomeTitle": "Home",
+  "pageHomeSearchButton": "Search Ride",
+  "pageHomeCreateButton": "Create Drive",
+
+  "rideEventPendingTitle": "Ride requested!",
+  "rideEventApprovedTitle": "Ride approved!",
+  "rideEventRejectedTitle": "Request rejected!",
+  "rideEventWithdrawnTitle": "Request withdrawn!",
+  "rideEventCancelledByDriverTitle": "Drive cancelled!",
+  "rideEventCancelledByRiderTitle": "Ride cancelled!",
+  "pageHomeUpcomingDriveToday": "Upcoming Drive today!",
+  "pageHomeUpcomingDriveTomorrow": "Upcoming Drive tomorrow!",
+  "pageHomeUpcomingRideToday": "Upcoming Ride today!",
+  "pageHomeUpcomingRideTomorrow": "Upcoming Ride tomorrow!",
+  "rideEventPendingMessage": "{username} has requested a ride",
+  "rideEventApprovedMessage": "{username} has approved your ride",
+  "rideEventRejectedMessage": "{username} has rejected your ride",
+  "rideEventWithdrawnMessage": "{username} has withdrawn his ride",
+  "rideEventCancelledByDriverMessage": "{username} has cancelled his drive",
+  "rideEventCancelledByRiderMessage": "{username} has cancelled his ride",
+  "pageHomeUpcomingTripMessage": "from {start} to {end} at {startTime}",
```

Comment:
This is the wrong way around. @istzen @fynsta is sorry, but this is because the parameters aren't given in the arb file. So the order of parameters is apparently chosen alphabetically. I think we have to include the lengthy placeholder syntax for multiple parameters